types/lunr/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for lunr.js 2.1
+// Type definitions for lunr.js 2.3
```

Comment:
The modified functions were introduced in 2.3.0 with this commit:

https://github.com/olivernn/lunr.js/commit/95d585298524cce52ab29c96f74ee8961d2ea9b8#diff-0fb61214341893489a1490be5f5b65a7